components/BidsTable.tsx

```diff
@@ -245,3 +260,39 @@ function jsNumberForAddress(address: string) {
   const seed = parseInt(addr, 16);
   return seed;
 }
+
+function PlaceBidIcon({ clientId }: { clientId: number }) {
```

Comment:
I'd call it `ClientIdIcon`. You can also move the `<a>` tag into this component, then it's very well incapsulated